mod/src/main/java/zsawyer/mods/mumblelink/mumble/UpdateData.java

```diff
@@ -131,9 +131,9 @@ public void set(Minecraft game) {
 
             // Position of the avatar
             fAvatarPosition = new float[]{
-                    Float.parseFloat(Double.toString(game.player.posX)),
-                    Float.parseFloat(Double.toString(game.player.posZ)),
-                    Float.parseFloat(Double.toString(game.player.posY))};
+                    Float.parseFloat(Double.toString(game.player.getPosition().getX())),
+                    Float.parseFloat(Double.toString(game.player.getPosition().getY())),

```

Comment:
This is probably wrong, Z and Y are switched in this PR. 
Unless they changed the coordinate system layout it will should cause strange attenuation effects.

Reply:
It's my fault ! I usually code by putting X, Y, Z

Reply:
Me too but in this case the array has a fixed order. So no way around it. ;-)